### DISCUSSION TITLE
Buefy already work with Bulma1.0.0?

### DISCUSSION THREAD

**Discussion by Krash0** [buefy/buefy#4087] [Q&A] [answered]
I tried to use Bulfy with the new Bulma 1.0.0 and I get errors with some undefined variables, no matter how I import Bulma the problem still happens...

bulma 1.0.0. is it supported?

**Comment by wesdevpro** [comment, marked as answer]
Not yet! See https://github.com/ntohq/buefy-next/issues/208
